rustls/examples/internal/bogo_shim.rs

```diff
@@ -6,6 +6,11 @@
 #[cfg(any(feature = "ring", feature = "aws_lc_rs"))]
 mod bogo_shim_impl;
 
+#[cfg(any(feature = "ring", feature = "aws_lc_rs"))]
+fn main() {
+    bogo_shim_impl::main();
+}
```

Comment:
Should this have been in the previous commit where the move was done and the quic feature removed? :thinking: